shoop_tests/core/test_categories.py

```diff
@@ -68,4 +68,4 @@ def test_category_wont_be_deleted():
     category.save()
     img.delete()
 
-    Category.objects.get(pk=1)
+    Category.objects.first()
```

Comment:
This change actually makes the test a no-op. Where `.get()` would have raised an ODNE when the object does not exist, this quietly returns None.

Change this to `Category.objects.get(pk=category.pk)` instead.